src/backend/chat.nim

```diff
@@ -88,7 +88,7 @@ proc sendImages*(chatId: string, images: var seq[string]): RpcResponse[JsonNode]
         "imagePath": image,
         # TODO is this still needed
         # "ensName": preferredUsername,
-        "text": "Please upgrade your status version to view images"
+        "text": "Image received"
```

Comment:
No, I meant sending the emoji instead of the text:

```suggestion
        "text": "🖼️"
```

Reply:
Should I worry about it if I expect that hardcoded text will be sent?

Reply:
Not sure what you're referring to with regards to worrying. But the idea is really just that we want to see the picture emoji in the notification if the message in question is an image.

Reply:
Hardcoded emoji... here's the result

![Screenshot 2023-01-11 at 17 24 38](https://user-images.githubusercontent.com/117639195/211845622-3cbcbdaa-a202-4dff-9f14-81860fecb6f1.jpg)

Reply:
Normal scenario

https://user-images.githubusercontent.com/117639195/211846660-721c3a1b-e484-4597-b771-60ba08350c62.mov

Reply:
I agree with the Image emoji as well

Reply:
So first, I wouldn't have expected to see the smile emoji in the notification. But then I realized that (I think) we send two messages for messages that have image + text. So it might be we're seeing the notification of the second message which is indeed the smile emoji text.

Reply:
You are right @0x-r4bbit . We send images and test in different status-go messages. If you were to send 4 images, it would be 4 different messages.